pkg/subctl/cmd/show_connections.go

```diff
@@ -73,9 +73,13 @@ func showConnections(cmd *cobra.Command, args []string) {
 
 func printConnections(connections []connectionStatus) {
 	template := "%-20s%-16s%-16s%-24s%-40s%-16s\n"
-
 	fmt.Printf(template, "GATEWAY", "CLUSTER", "REMOTE IP", "CABLE DRIVER", "SUBNETS", "STATUS")
 
+	if len(connections) == 0 {
+		fmt.Println("no resource found")
```

Comment:
To match `kubectl`,

```suggestion
		fmt.Println("No resources found.")
```

(and likewise for the other two occurrences).

Reply:
One more question - should the header line be printed in or not in the case of no resources?

Reply:
> One more question - should the header line be printed in or not in the case of no resources?

`kubectl` doesn’t print the header, so no (see `kubectl get pods -n default` on a newly-deployed cluster for example).